### PR TITLE
Move explicit pliers/multitool requirements to WRENCH 1 requirement

### PR DESCRIPTION
**Pull request by TheShadowFerret**
#### Summary
None

#### Purpose of change

(Slip-joint) pliers or multitools have no magic qualities that other bolt turning tools do not have, especially when the regular and tongue-and-groove pliers both say  ``Anything too complex will require a wrench``.

#### Describe the solution

Move all "only pliers or multitool" requirements to WRENCH 1.

#### Describe alternatives you've considered

Giving boltcutters WRENCH 1, and then moving all of those over too.

#### Testing

Load ingame.

#### Additional context

image

~Demonstration + what's left that still explicitly calls for boltcutters

### CODE REVIEW

File: data/json/construction.json [hunk 1]
```diff
@@ -3496,8 +3496,7 @@
     "category": "APPLIANCE",
     "required_skills": [ [ "fabrication", 0 ] ],
     "time": "0 s",
-    "qualities": [ { "id": "CUT", "level": 2 } ],
-    "tools": [ [ [ "pliers", -1 ], [ "multitool", -1 ] ] ],
+    "qualities": [ { "id": "CUT", "level": 2 }, { "id": "WRENCH", "level": 1 } ],
```
Comment (Fris0uman) [inline data/json/construction.json:3499]: but a wrench would be useless to uncover wires in a wall

Comment (TheShadowFerret) [reply data/json/construction.json:3499]:
```suggestion
    "qualities": [ { "id": "CUT", "level": 2 } ],
    "tools": [ [ [ "pliers", -1 ], [ "multitool", -1 ] ] ],
```

**Comment by Karlington2** [comment]
Am I missing something, or did this delete the recipe for the "basic" chain links?

**Comment by RandDenner** [comment]
Seems to have, yeah.

**Comment by RandDenner** [comment]
I put it back in PR #57200

**Pull request merged**